moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -802,7 +767,9 @@ const ProjectComponentsMapEdit = ({
                 aria-describedby="alert-dialog-description"
               >
                 <DialogTitle id="alert-dialog-title">
-                  <h2>{"Delete Component?"}</h2>
+                  <div>
```

Comment:
Forgot to note that this was to squash a console warning about nested `<h2>` tags (`DialogTitle` returns a `<h2>`)